test/integration/alchemy.test.ts

```diff
@@ -17,6 +24,7 @@ describe('E2E integration tests', () => {
     describe('AlchemyProvider', () => {
       function testNetwork(network: Network) {
         it(`get blockNumber on ${network}`, async () => {
+          console.log('testing network', network);
```

Comment:
Nit: remove this log?

Reply:
wanted to leave it there you can tell which network it's on since it takes a long time to finish